### PR TITLE
Type selection will be disabled in case of one or more code files

### LINKED ISSUE #847: Disable analysis type select after first importing/code creating
To avoid cases of importing different designs type in the same place
image-2021-06-09-17-51-45-169

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #847

### CODE REVIEW

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx [hunk 1]
```diff
@@ -59,7 +62,7 @@ function AnalysisSettings(props) {
             modifiers: { uneditable: !isEditable } 
           })}
         >
-            {isEditable ?
+            {isTypeEditable ?
```
Comment (dmitrys-odysseus) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx:65]: Why do we now ignore "isEditable"? Shouldn't it be isEditable && "isTypeEditable"?

Comment (ssuvorov-fls) [reply src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx:65]: "isEditable" is used in "isTypeEditable" initializtion

Comment (dmitrys-odysseus) [reply src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx:65]: right, missed that

File: src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx [hunk 2]
```diff
@@ -36,11 +36,14 @@ function AnalysisSettings(props) {
     typeSelected,
     setType,
 
+    files,
     isCodeLocked,
     canLockCode,
     codeLock,
   } = props;
 
+  const isTypeEditable = files.length > 0 ? false : isEditable;
```
Comment (dmitrys-odysseus) [inline src/modules/AnalysisExecution/components/ViewEditAnalysis/Settings/presenter.jsx:45]: Could be shortened to `files.length > 0 && isEditable`

**Review by dmitrys-odysseus — APPROVED** [review]

**Pull request merged**